src/app/static/src/app/components/downloadResults/DownloadResults.vue

```diff
@@ -2,47 +2,17 @@
     <div class="container">
         <div class="row">
             <div class="col-sm">
-                <div id="spectrum-download">
-                    <download :translate-key="translation.spectrum"
-                              @trigger-download="downloadSpectrumOutput"
-                              :disabled="!spectrum.downloadId || spectrum.preparing"
-                              :file="spectrum"/>
-                    <div class="pb-2">
-                    <error-alert v-if="spectrum.downloadError" :error="spectrum.downloadError"></error-alert>
+                <template v-for="type in Object.values(DownloadType)" :key="type">
+                    <div :id="`${type}-download`" v-if="switches[type]">
+                        <download :translate-key="`${type}Download`"
```

Comment:
I wonder if we should put this into the `downloadConfig.ts` like the `switches` a map of type to the key we want to use. I do like all the download stuff being in the single config file.

Reply:
hmmm do you mean because theres duplication between this and the `locales.ts` type? because otherwise everything is auto generated, also the types in the locales file ensures we have the matching translation key

Reply:
No, only that when you're adding a new file type you have to look here to see what the key is. It is `{type}Download" instead of just being able to see the full string in the `downloadConfig.ts` ? That make sense?